accounts-db/src/accounts.rs

```diff
@@ -561,48 +509,6 @@ impl Accounts {
         self.accounts_db.store_uncached(slot, &[(pubkey, account)]);
     }
 
-    fn lock_account<'a>(
-        &self,
-        account_locks: &mut AccountLocks,
-        keys: impl Iterator<Item = (&'a Pubkey, bool)> + Clone,
-    ) -> Result<()> {
-        for (k, writable) in keys.clone() {
-            if writable {
-                if account_locks.is_locked_write(k) || account_locks.is_locked_readonly(k) {
-                    debug!("Writable account in use: {:?}", k);
-                    return Err(TransactionError::AccountInUse);
-                }
-            } else if account_locks.is_locked_write(k) {
-                debug!("Read-only account in use: {:?}", k);
-                return Err(TransactionError::AccountInUse);
-            }
-        }
-
-        for (k, writable) in keys {
-            if writable {
-                account_locks.write_locks.insert(*k);
-            } else if !account_locks.lock_readonly(k) {
-                account_locks.insert_new_readonly(k);
-            }
-        }
-
-        Ok(())
-    }
-
-    fn unlock_account<'a>(
```

Comment:
these functions used to be on `Accounts` but don't even use `self`.
They are no on the `AccountLocks` struct directly.